docs/source/monitors.rst

```diff
@@ -89,8 +91,7 @@ Here is an example of how to configure a new monitor suite in your project:
 
   ``name`` suite name
 
-  ``monitors`` list of :class:`~spidermon.core.monitors.Monitor` that will be executed
-  if this suite is enabled.
+  ``monitors`` list of :class:`spidermon.core.monitors.Monitor` that will be executed if this suite is enabled.
```

Comment:
This change should now be reverted.

Reply:
Roger that, reverted the change. Did wanted to ask what is the `~` for? Is it something that completes the relative path of the class that we are trying to list out.